app/src/main/res/values/strings.xml

```diff
@@ -555,4 +555,9 @@ be temporary, and you can try again later.</li>
     <string name="gv_prompt_username_hint">Username</string>
     <string name="gv_prompt_password_hint">Password</string>
     <string name="gv_prompt_clear">Clear</string>
+
+    <string name="secure_connection">Secure Connection</string>
+    <string name="insecure_connection">Insecure Connection</string>
+    <string name="security_verified">Verified by: %1$s</string>
+    <string name="site_security">Site Security</string>
```

Comment:
Can you add some comments here explaining what the Strings are used for - or where they show up? Comments *before* a String definition will show up in the tools our translators use and help them to come up with a good translation.

nit: I wonder if we should use a prefix here, just to group the strings, e.g. security_popup_*